src/pages/guidelines/color/implementation.mdx

```diff
@@ -0,0 +1,546 @@
+above the tabs; it uses `$background` from the base set. The tab component is
+layered on top of the page background to create the first layer. The selected
+tab uses `$layer-01` and the unselected tabs use `$layer-accent-01` which is not
+considered a proper layer but a supporting color for `$layer` inside of
+components. The tab content area attached to the selected tab is also only one
+layer above the base and so also uses `$layer-01` as its background.
+
+In the tab's main section, the text input field is placed on top of `$layer-01`
+making it a part of the next layer level and will use tokens from the 02 layer
+set, so `$field-02` and `$border-strong-02`. Also a part of the second layer
+level are the tiles in the sub-section, this includes the tile background
+`$layer-02` as well as the border between the tiles `$border-subtle-02`.
+However, components added on top of the tiles—the text input and overflow
+menu—are considered part of third layer level and will use tokens from the 03
+layer set.
+
+Token key
+
+- Base set: Magenta
+- Layer set 01: Purple
+- Layer set 02: Teal
+- Layer set 03: Cyan
+
+<Row>
+<Column colLg={12}>
+
+<Tabs>
+
+<Tab label="White">
+
+![In context layer set token example in the White theme](images/color-implementation-layer-set-2-W.png)
+
+</Tab>
+
+<Tab label="Gray 10">
+
+![In context layer set token example in the Gray 10 theme](images/color-implementation-layer-set-2-G10.png)
+
+</Tab>
+
+<Tab label="Gray 90">
+
+![In context layer set token example in the Gray 90 theme](images/color-implementation-layer-set-2-G90.png)
+
+</Tab>
+
+<Tab label="Gray 100">
+
+![In context layer set token example in the Gray 100 theme](images/color-implementation-layer-set-2-G100.png)
+
+</Tab>
+
+</Tabs>
+
+</Column>
+</Row>
+
+<br />
+
+#### Building components with layer tokens
+
+Building component with layer tokens works much like how you would have built
+component variants in v10. For each layer level that a component lives on, a
+separate component variation must be built using the explicit layer set tokens.
+In v10, these color variants were known as the `light` prop variants, however
+these new layering tokens allows for a three levels color variant in components
+that wasn't possible in v10.
+
+Spec each component variant with it corresponding layer set tokens. For elements
+that are not part of the layering model like type or icons use the normal color
+tokens as you would have in v10. These tokens will be the same across variants
+as they are high enough contrast not to need to change with each layer.
+
+<Row>
+<Column colLg={8}>
+
+![Component spec example using layering tokens](images/color-implementation-layer-set-3.png)
+
+</Column>
+</Row>
+
+<Caption>
+  Example component spec in the white theme using the layer set 01 tokens.
+</Caption>
+
+### Contextual layer tokens
+
+Contextual tokens are smart tokens used to automatically map the layering model
```

Comment:
Calling this new thing both contextual tokens and smart tokens can be confusing given that we don't really say what they are. Maybe we could describe contextual tokens as:

> Contextual tokens are a special case of design token where the value changes depending on where it is used on a page.

And then refer to them as only contextual tokens throughout the page?

Reply:
"smart tokens" was more a metaphor then an actual term I was using to help people understand how they're different than explicit tokens. Jan used this term a lot when talking about them so I thought it was a safe term.

Reply:
I could definitely see how they're helpful. It seems like in my reading I thought of contextual tokens as the same as smart tokens which made the sentence feel redundant. However, it seems like you're meaning for it to be read is that contextual tokens are a type of smart token and that smart tokens are a broader group than contextual tokens.